### PR TITLE
Add vendor and model id of STLink V2 to USBDebugger resource

### PR DESCRIPTION
**Pull request by MischaZihler**
**Description**
This PR adds STLink V2 vendor and model to the USBDebugger resource. The code has been tested on a distributed labgrid testing system.

**Checklist**
- [ ] Documentation for the feature
- [ ] Tests for the feature 

- [ ] The arguments and description in doc/configuration.rst have been updated

- [ ] Add a section on how to use the feature to doc/usage.rst

- [ ] Add a section on how to use the feature to doc/development.rst

- [x] PR has been tested

- [ ] Man pages have been regenerated

### CODE REVIEW

**Comment by Bastian-Krause** [comment]
Also: please add a Signed-off-by to your commit message, see [our contribution docs](https://labgrid.readthedocs.io/en/latest/development.html#developer-s-certificate-of-origin).

File: labgrid/resource/udev.py [hunk 1]
```diff
@@ -8,9 +8,9 @@
 import attr
 
 from ..factory import target_factory
-from .common import ManagedResource, ResourceManager
-from .base import SerialPort, NetworkInterface
 from ..util import Timeout
+from .base import NetworkInterface, SerialPort
+from .common import ManagedResource, ResourceManager
```
[the remark this answers is not in the document]
Comment (MischaZihler) [reply labgrid/resource/udev.py]: hi @Bastian-Krause, see my new commit

**Comment by Bastian-Krause** [comment]
The DCO check is [still not happy](https://github.com/labgrid-project/labgrid/pull/1193/checks?check_run_id=13887645388). Looks like a mix up of email addresses. The author email address should match the Signed-off-by address.

**Review by Bastian-Krause — APPROVED** [review]
Thanks!

**Pull request merged**